Minor edits to transmission text (#656)
greenelab/covid19-review: content/08.transmission.md

While droplet-based and contact transmission were initially considered to be the primary modes by which SARS-CoV-2 spread [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, the possibility of aerosol transmission has also been raised [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939].
For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could be spreading via aerosols [@doi:10.1038/s41586-020-2271-3].
The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV [@doi:10.1056/NEJMc2004973], and fomite transmission could also play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).
However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458], and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].

> These type of transmission are likely to contribute, to varying extents, to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.

Reviewer: there are just so many `to`s in a row here. It's fine but maybe consider a light reword?

Author: Hahaha in retrospect, also some casual tautology?
```suggestion
These mechanisms may differ in their relevance to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.
```

Reviewer: 👍
